di-ipv-ticf-stub/lambdas/src/services/ticfService.ts

```diff
@@ -49,12 +50,15 @@ export async function processGetVCRequest(
 
   const returnJwt = await signJwt(payload, ticfSigningKey);
   return {
-    sub: ticfRequest.sub,
-    govuk_signin_journey_id: ticfRequest.govuk_signin_journey_id,
-    vtr: ticfRequest.vtr,
-    vot: ticfRequest.vot,
-    vtm: ticfRequest.vtm,
-    "https://vocab.account.gov.uk/v1/credentialJWT": [returnJwt],
+    response: {
```

Comment:
I think you need to change the return type of the method to match this change.

Have you set up TS correctly in VS Code? It should warn you pretty quickly about this. I'm also a bit surprised it doesn't get flagged in the pre-merge checks, I wonder if we need to add something to `pre-merge-checks.yml` for this (perhaps as a separate ticket, if it's not quick).

Reply:
Method (processGetVCRequest) been changed to return (Promise<ServiceResponse>).

Reply:
Sounds good (although I can't see the change yet?)

Reply:
VScode missing the autosave, now u'll see.